packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts

```diff
@@ -101,14 +101,14 @@ export class QuorumTestLedger implements ITestLedger {
   }
 
   public async getFileContents(filePath: string): Promise<string> {
-    const response: any = await this.getContainer().getArchive({
+    const response: unknown = await this.getContainer().getArchive({
```

Comment:
@yayoiukai2021 

I recommend simply omitting the explicit type declaration here in order to make the linter warning go away. Using unkown in this particular case just makes the compiler fail somewhere down the line.

```suggestion
const response = await this.getContainer().getArchive({
```